src/pages/home/home.ts

```diff
@@ -1,45 +1,132 @@
-import { Component } from '@angular/core';
-import { NavController, Platform } from 'ionic-angular';
+import { Component } from '@angular/core/';
+import {
+  GoogleMaps,
+  GoogleMap,
+  GoogleMapsEvent,
+  GoogleMapOptions,
+  CameraPosition,
+  MarkerOptions,
+  Marker,
+} from '@ionic-native/google-maps';
+import { Platform } from 'ionic-angular';
 import { YelpService } from '../../app/services/yelp/yelp.service';
 import { Observable } from 'rxjs/Observable';
 import { Geolocation } from '@ionic-native/geolocation';
 import { WorkfromService } from '../../app/services/workfrom/workfrom.service';
 
 @Component({
   selector: 'page-home',
-  templateUrl: 'home.html'
+  templateUrl: 'home.html',
 })
 export class HomePage {
-
   yelpLocations$: Observable<any>;
   workfromLocations$: Observable<any>;
 
   latitude: number;
   longitude: number;
 
+  map: GoogleMap;
+  mapElement: HTMLElement;
+
   constructor(
-    navCtrl: NavController,
-    platform: Platform,
+    private googleMaps: GoogleMaps,
+    public platform: Platform,
```

Comment:
I tried removing the keyword before and it would give me an error referencing it outside of the constructor. I tried again just to make sure and It resolves with the same error saying it doesn't know where 'platform' comes from, so I'm going to leave it as is for now.